api/tests/files/invoice/rendered_invoice.html

```diff
@@ -406,8 +406,10 @@ <h3>Montant des remboursements par lieu et détail de la part individuelle et co
             <th>Incidents (TTC)</th>
             <th>Contribution offreur incidents (TTC)</th>
             <th class="coloredSection">Montant remboursé (TTC)</th>
+
```

Comment:
Détail : Pour éviter ces lignes vides, tu pouvais ajouter des `+` dans le modèle Jinja :
`{%+ if not is_caledonian_invoice %}`
`{%+ endif %}`